controllers/discovery/initDiscoveryDomainData.js

```diff
@@ -45,11 +51,21 @@ const InitDiscoveryDomainData = async (req, res) => {
     });
     let suggestedDomains = domainWithCommonFollowerResult;
 
+    let totalData = await sequelize.query(totalDataQuery, {
+      type: QueryTypes.SELECT,
+      replacements: {
+        userId
+      },
+      raw: true
+    });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the `userId` is being passed as a replacement for the `totalDataQuery`, but it doesn't seem like the `userId` is used anywhere in that query. This seems unnecessary and could be removed unless there are plans to use it in the future.
```diff
-       replacements: {
-         userId
-       },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->